examples/bastion/readme.md

```diff
@@ -1,66 +1,96 @@
->Note: The config_no_pub_ip.json deploys an Azure Bastion, VNET and a jumpbox (no pub IP), then you can login to the jumpbox via the azure bastion and deploy the rest of your azurehpc deployment. The config_no_pub_ip.json contains an example of using cloud-init in AzureHPC. The AzureHPC prerequisites are installed on the jumpbox (with no public IP) using a cloud-init script (cloud-init.txt (Installs git, jq, AzureHPC git clone, azcopy and azcli). The Cloud-init script needs to be either uploaded to blob storage in advance or referenced in the config file by @cloud-init.txt (read-in directly).
+The Linux jumpbox supports CentOS and Ubuntu images.
 
-## Initialise the project
+## Step 1 - Install and initialize AzureHPC
 
-To start you need to copy this directory and update the `config.json`.  Azurehpc provides the `azhpc-init` command that can help here by compying the directory and substituting the unset variables.  First run with the `-s` parameter to see which variables need to be set:
+Clone the `azhpc` repository and source the `install.sh` script.
 
 ```
-azhpc-init -c $azhpc_dir/examples/bastion -d bastion -s
+git clone https://github.com/Azure/azurehpc.git
+source azurehpc/install.sh
 ```
 
-The variables can be set with the `-v` option where variables are comma separated.  The output from the previous command as a starting point.  The `-d` option is required and will create a new directory name for you.  Please update to whatever `resource_group` you would like to deploy to:
+## Step 2 - Initialize the project
 
-```
-azhpc-init -c $azhpc_dir/examples/bastion -d bastion -v resource_group=azurehpc-cluster
-```
+To start you need to copy this directory in the desired working location and update the `variables.json` file with the desired parameters.
+
+| Variable                     | Value                                                                   |
+|------------------------------|-------------------------------------------------------------------------|
+| **resource_group**           | The resource group to put the resources                                 |
+| **location**                 | Azure region to deploy resources                                        |
+| **vnet_ip_range**            | IP address range in CIDR notation for Bastion VNet                      |
+| **default_subnet_ip_range**  | IP address range in CIDR notation for VMs subnet                        |
+| **bastion_subnet_ip_range**  | IP address range in CIDR notation for Bastion subnet                    |
+| **jumpbox_image**            | CentOS or Ubuntu marketplace image for Linux jumpbox                    |
+| **key_vault**                | Unique name to assign to Key Vault                                      |
+| **secret_name**              | **DO NOT MODIFY** - Name of the secret storing Windows VM user password |
+
+Then run the `init.sh` script to automatically create the `prereqs.json` and `config.json` configuration files.

```

Comment:
Would it be clearer to also highlight the init.sh step, showing the exact command?